k8s/helm/templates/deployment.yaml

```diff
@@ -89,6 +89,10 @@ spec:
               value: "{{- .Values.googleTagManagerPreview }}"
             - name: COOKIE_SETTINGS_URL
               value: "{{- .Values.cookieSettingsUrl }}"
+            - name: CDN_URL
```

Comment:
While the defaulted cdn.url (and assets_path) are correctly being injected, does the deploy_app.sh need extending to allow these to be overridden?